src/apps/filters/h3.c

```diff
@@ -140,6 +190,9 @@ int main(int argc, char *argv[]) {
     if (has("latLngToCell", 1, argv) && latLngToCellCmd(argc, argv)) {
         return 0;
     }
+    if (has("cellToBoundary", 1, argv) && cellToBoundaryCmd(argc, argv)) {
+        return 0;
```

Comment:
As above, I would have gone with `return cellToBoundaryCmd(argc, argv)` and have each command return `H3_ERROR`

Reply:
I agree with you, but would like that to also be a separate PR.